README.md

```diff
@@ -6,16 +6,16 @@ Simple benchmark for testing your DOM diffing algorithm.
 
 | Position | Library | Size | Speed |
 | --- | --- | --- | --- |
-| 🏆 1 | [udomdiff](https://github.com/WebReflection/udomdiff) | 397B | ~248ms |
-| 2 | [snabbdom](https://github.com/snabbdom/snabbdom) | 412B | ~253ms |
-| 3 | [list-difference](https://github.com/paldepind/list-difference/) | 281B | ~258ms |
-| 4 | [spect](https://github.com/spectjs/spect) | 218B | ~269ms |
-| 5 | [stage0](https://github.com/Freak613/stage0) | 941B | ~326ms |
-| 6 | [heckel](https://johnresig.com/projects/javascript-diff-algorithm/) | 449B | ~481ms |
+| 🏆 1 | [udomdiff](https://github.com/WebReflection/udomdiff) | 397B | ~20ms |
+| 2 | [list-difference](https://github.com/paldepind/list-difference/) | 281B | ~24ms |
+| 3 | [heckel](https://johnresig.com/projects/javascript-diff-algorithm/) | 449B | ~39ms |
```

Comment:
strange heckel is 3, with all those extra operations

Reply:
Well, the improvement on the _Dommy_ side maybe affected results, and I'm also not sure my laptop gives similar results compared to yours ... maybe you can run the bench a few time and provide an average?

Reply:
cool, I almost consistently have heckel on 4 with latest master.

I'll add changes on this branch

<img width="1000" alt="Screen Shot 2020-04-18 at 11 23 55 AM" src="https://user-images.githubusercontent.com/360826/79641727-4d7eed80-8167-11ea-82fa-53037cf82b35.png">

Reply:
Also worth saying that *maybe* we should incrementally push all results and provide a total of just that, as I'm not sure myself why numbers differ so much between single benchmarks, and the total amount of time spent to run the whole thing ... although, I believe that's relevant too, but maybe not super fair ... trying to think what could be the bottleneck

Reply:
OK, this LGTM

Reply:
all results would be problematic too I think, the libs would need versions and it would differ from machine to machine, node version.

maybe run 3 times randomized by library and take some average?